ssh-add.desktop

```diff
@@ -0,0 +1,4 @@
+[Desktop Entry]
+Name=ssh-add
+Exec=ssh-add
+Type=Application
```

Comment:
What happens if you have password-protected keys.

Does ssh-add hang asking for a passphrase?